docs/help/submit-issue.md

```diff
@@ -0,0 +1,108 @@
+
+# Submit an Issue
+
+## Where to Find Help
+
+When you run into issues while building or using the Trio app, follow the steps below to get help efficiently and show respect for the time our volunteers generously offer.
+
+-  [ ] Always start by searching [**TrioDocs**](https://triodocs.org) — just use the 🔎 search bar at the top.
+- [ ] Search your favorite support group (Facebook or Discord):  
+      
+    !!! tip "Help the Community Help You"
+        
+        When you run into issues building or using the _Trio_ app, there’s a **wonderful community of loopers** ready to help. These volunteers offer their time freely — no one is paid to support or troubleshoot your setup. They simply want to help others.  
+        To respect their efforts and get the most helpful response, please do your homework first and **include the key details they’ll need when asking for help**.
+      
+    - **[Facebook](https://facebook.triodocs.org)**
+        -  Not sure **how to search in** a **Facebook** group? [Here’s a video to help.](https://www.youtube.com/watch?v=_vSN6C-Uo04)  
+        - If you use **Facebook**, check the **Featured posts** at the top of the group, where many common questions have already been answered.
+    - **[Discord](https://discord.triodocs.org)**: if your issue is related to:
+        - [Build with Mac](https://discord.com/channels/1020905149037813862/1239982577008377958)
+        - [Build with Browser](https://discord.com/channels/1020905149037813862/1239982525820960860)
+        - [Trio Settings](https://discord.com/channels/1020905149037813862/1239982177190678549)  
+      
+- [ ] **Avoid posting the same question in multiple groups.**  
+    Many mentors follow several communities, and cross-posting can lead to confusion. Only try another group if you've received no replies after several hours.
+
+    !!! important "Do not cross-post your message"
+      
+        Please, do **not** send the same message on multiple sites/channels.  
+        Our volunteers monitor multiple sites. 
+
+- [ ] If your **question hasn’t been answered** by checking *TrioDocs* or browsing the *Featured posts* in the Facebook group, go ahead and **post your question**.  
```

Comment:
I think lines 32 through 38 belong in the details in How to Ask for Help.

The details currently in How to Ask for Help are specifically about asking for Build Help - and seem to be redundant with some details in What to Share . . .

Reply:
✔️ Addressed in f2df279